crates/uv/src/commands/venv.rs

```diff
@@ -134,10 +131,6 @@ async fn venv_impl(
     printer: Printer,
     relocatable: bool,
 ) -> miette::Result<ExitStatus> {
-    if preview.is_disabled() && relocatable {
```

Comment:
does this mean that relocatable is now considered a stable API? (i'd be delighted with this -- not sure if i am right to make this inference, though)

Reply:
I guess so! I tore all of the preview labeling out pretty mechanically. cc @charliermarsh 

We can add an entry to the changelog if we're on the same page.